README.md

```diff
@@ -31,7 +31,13 @@ the tests, and run them individually.
 ### Backend Notes
 
 - All endpoints should be behind an `api/` path
-
+- Access to Auth0 Management API requires configuation in `.env` (which is in `.gitignore`)
+  - Login to http://manage.auth0.com
+  - Navigate to `APIs` -> `Auth0 Management API` -> `Auth0 Management API (Test Application)`
+  - Get `Client ID` and `Client Secret` values
+  - assign to following variables in `.env`
+    - `AAUTH0_TOKEN_ACCESS_CLIENT_ID`
```

Comment:
two A's?

Also, after going to the `Auth0 Management API` ya need to click the `Machine to Machine Applications` tab.

also, what scopes are needed in case we need to create a new token?
looks like you just have `read:users`, `read:users_app_metadata`

Reply:
Will fix the to A's.  Sigh...
Will document the scopes.